client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -296,7 +296,7 @@ const CancelPurchaseRefundInformation = ( {
 		);
 	} else {
 		text = i18n.translate(
-			"When you cancel your subscription, you'll be able to use %(productName)s until your subscription expires. " +
+			"You'll be able to use %(productName)s until your subscription expires. " +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'When you cancel your subscription, you'll be able to use %(productName)s until your subscription expires. Once it expires, it will be automatically removed from your site.' )` *ES Score: 8*